srcpkgs/applet-window-buttons/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'applet-window-buttons'
+pkgname=applet-window-buttons
+version=0.8.1
+revision=1
+build_style=cmake
+hostmakedepends="extra-cmake-modules gettext qt5-host-tools qt5-qmake
+kdeplasma-addons5 plasma-framework-devel"
+makedepends="plasma-framework-devel kdecoration-devel qt5-devel"
+depends="plasma-workspace"
```

Comment:
I don't like having plugins depending on the application they are a plugin for... Maybe remove the dependency?